include/variant_parser/variant_record.hpp

```diff
@@ -0,0 +1,193 @@
+#include <seqan3/io/stream/concept.hpp>
+#include <fstream>
+#include <map>
+
+class info_entry
+{
+public:
+    info_entry(std::string keyi, std::uint8_t numberi, std::string typei, std::string descriptioni, std::string sourcei, std::string versioni) :
+        key{std::move(keyi)}, number{numberi}, type{std::move(typei)}, description{std::move(descriptioni)}, source{std::move(sourcei)}, version{std::move(versioni)}
+    {}
+    std::string key{};
+    std::uint8_t number{};
+    std::string type{};
+    std::string description{};
+    std::string source{};
+    std::string version{};
+};
+
+class variant_header
+{
+public:
+    variant_header() {}
+
+    /*! \brief Set the file format for this VCF file header.
+     *
+     *  \param fileformati The input fileformat.
+     */
+    void set_fileformat(std::string fileformati)
+    {
+        fileformat = fileformati;
+    }
```

Comment:
Just an idea, so it doesn't look like a typo:
```suggestion
    /*! \brief Set the file format for this VCF file header.
     *
     *  \param fileformat_i The input fileformat.
     */
    void set_fileformat(std::string fileformat_i)
    {
        fileformat = fileformat_i;
    }
```